receive_message_queue_job.py

```diff
@@ -36,6 +37,16 @@ async def main():
                     if (response.status_code == 200):
                         await receiver.complete_message(msg)
                         logging.info("Current msg has been deleted")
+                elif "pusher" in json.loads(str(msg)):
+                    logging.info("Current msg: %s", json.loads(str(msg)))
+                    print(json.loads(str(msg))) 
+                    response = requests.post(url=CONNECTION_URL + "/github-webhook/", data=str(msg), headers=headers)
```

Comment:
this is currently using the wrong event, but will be fixed with:
```suggestion
                    headers = {'Content-type': 'application/json', 'X-GitHub-Event':'push'}
                    response = requests.post(url=CONNECTION_URL + "/github-webhook/", data=str(msg), headers=headers)
```